### ISSUE TITLE
Support for JSON schema draft 2020-12, 2019-09

### ISSUE THREAD

**Issue by marcelhanser** [Apicurio/apicurito#156]
Hi apicurio team

i tried to create/upload an artifact with following JSON schema
```

{
  "$id": "https://lalal.com/v2.test.json",
  "$schema": "https://json-schema.org/draft/2019-09/schema",
  "description": "A representation of a person, company, organization, or place",
  "type": "object"
}

```
and got the error:
`Caused by: com.networknt.schema.JsonSchemaException: Unknown MetaSchema: https://json-schema.org/draft/2019-09/schema`
so i guess 2020-12, 2019-09 is not supported.

Are there any plans to support them soon?

Best & Thx
Marcel

**Comment by EricWittmann** [comment]
Yes we are a little behind in our JSON schema support and this needs to be improved.

Note: you've opened this GH issue against the apicurito repository but I assume it's an issue with Apicurio Registry.  I'm going to move the issue to that repo.
